File: PSW-projekat/hospital-be-main/src/HospitalLibrary/Core/Model/Prescription.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HospitalLibrary.Core.Model
{
    public class Prescription : BaseModel
    {
        public virtual ICollection<Medicine> Medicines { get; set; }
        public String Description { get; set; }

        public Prescription() { }

        public Prescription(ICollection<Medicine> medicines, string description)
        {
            Medicines = medicines;
            Description = description;
        }
    }
}

File: PSW-projekat/PZL-Angular/src/app/modules/hospital/model/prescription.ts
import { Appointment } from "./appointment.model";
import { Medicine } from "./medicine";
import { Symptom } from "./symptom";

export class Prescription {
    id: number;
    deleted: boolean;
    medicines: Medicine[];
    description: string;

    public constructor(id: any, deleted: any, medicines: any, description: any) {
        this.id = id;
        this.deleted = deleted;
        this.medicines = medicines;
        this.description = description;

    }
}